### PR TITLE
changelog and version increase to 0.2.0

### PR DESCRIPTION
**Pull request by psychocoderHPC**
- The change-log is not containing all PR's, it is a collection of the most important changes for the user.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+0.2.0 "Khan"
+------------
+**Date:** 2019-02-XX
+- update alpaka to 0.4.0 #128 #143
+- support `cudaGetErrorName` #113
+- support `cudaEventBlockingSync` #98
+- support for alpaka OMP4 backend #137
+- support for alpaka TBB backend #105
+- support for alpaka HIP backend #97
+- adds atomicAnd, atomicXor, atomicOr #117
```
Comment (ax3l) [inline CHANGELOG.md]:
```suggestion
- adds `atomicAnd`, `atomicXor`, `atomicOr` #117
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+- configuration header/cupla support without CMake #112
+- move all accelerator-specific code to an inline namespace #106
+- pin memory allocated with `cuplaMallocHost` #144
+- switch for dependency alpaka from submodule to subtree #103
```
Comment (ax3l) [inline CHANGELOG.md]:
```suggestion
- switch internally shipped version of alpaka from a git submodule to subtree #103
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,23 @@
 Changelog
 =========
 
+0.2.0 "Khan"
+------------
+**Date:** 2019-02-XX
```
Comment (ax3l) [inline CHANGELOG.md]: don't forget this :)

Comment (psychocoderHPC) [reply CHANGELOG.md]: Yes the date is currently not clear but very soon it is.

Comment (psychocoderHPC) [reply CHANGELOG.md]: I will set the release date to next Monday 2019-02-10

**Comment by psychocoderHPC** [comment]
@ax3l thx for the review and the suggestions!

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,24 @@
...
+- support for alpaka OMP4 backend #137
+- support for alpaka TBB backend #105
+- support for alpaka HIP backend #97
+- adds `atomicAnd`, `atomicXor`, `atomicOr` #117
+- configuration header/cupla support without CMake #112
+- move all accelerator-specific code to an inline namespace #106
+- pin memory allocated with `cuplaMallocHost` #144
+- switch internally shipped version of alpaka from a git submodule to subtree #103
+- refactor cupla kernel execution #136
+- compile cupla interfaces into static cupla library #140
```
Comment (sbastrakov) [inline CHANGELOG.md]: Perhaps replace `into static cupla library` with `into a static library`?

File: cuplaConfig.cmake [hunk 5]
```diff
@@ -273,7 +273,7 @@ endif()
 # Find cupla version.
 ################################################################################
 # FIXME: Add a version.hpp
```
Comment (sbastrakov) [inline cuplaConfig.cmake]: This is fixed, maybe remove the line?

Comment (psychocoderHPC) [reply cuplaConfig.cmake]: In cupla we have currently no version.hpp file

Comment (sbastrakov) [reply cuplaConfig.cmake]: It is linked above xd

Comment (psychocoderHPC) [reply cuplaConfig.cmake]: ohh thanks so I need to change the version there too.

Comment (sbastrakov) [reply cuplaConfig.cmake]: Yes, as I wrote in the review above.

**Comment by psychocoderHPC** [comment]
I change the date to 11.02.2020

**Comment by ax3l** [comment]
Can go in as well, @psychocoderHPC don't forget to release on Tuesday :D (or just update the date otherwise).

**Pull request merged**